openedge.properties

```diff
@@ -1,26 +1,18 @@
 category=Languages
 description=Code Analyzer for OpenEdge
 homepageUrl=https://github.com/Riverside-Software/sonar-openedge
-archivedVersions=2.14.4,2.19.2
-publicVersions=2.24.2
+archivedVersions=2.19.2,2.24.2
+publicVersions=
 
 defaults.mavenGroupId=eu.rssw.sonar.openedge
 defaults.mavenArtifactId=sonar-openedge-plugin
 
-2.14.4.description=New expression engine in ABL Parser
-2.14.4.sqVersions=[8.9,9.8]
-2.14.4.date=2021-11-23
-2.14.4.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.14.4
-2.14.4.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.14.4/sonar-openedge-plugin-2.14.4.jar
-
 2.19.2.description=Version 2.19.2
-2.19.2.sqVersions=[8.9,9.9.*]
+2.19.2.sqVersions=[9.9,9.9.*]
 2.19.2.date=2023-01-10
 2.19.2.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.19.1
 2.19.2.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.19.1/sonar-openedge-plugin-2.19.2.jar
 
 2.24.2.description=Version 2.24.2
-2.24.2.sqVersions=[9.9,LATEST]
+2.24.2.sqVersions=[9.9,10.3]
```

Comment:
Is there a reason for removing the links for 2.24.2 and archiving it?

Reply:
Catching this up with another change already in Master. The maintainer removed the binary.